src/formatDate.js

```diff
@@ -8,7 +8,26 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldDateFormat = date.split(fromFormat[fromFormat.length - 1]);
+  const newDateFormat = [];
+
+  for (let i = 0; i < oldDateFormat.length; i++) {
+    if (toFormat.indexOf(fromFormat[i]) === -1) {
+      if (fromFormat[i].length === 2) {
+        if (+oldDateFormat[i] < 30) {
+          newDateFormat[toFormat.indexOf('YYYY')] = '20' + oldDateFormat[i];
+        } else {
+          newDateFormat[toFormat.indexOf('YYYY')] = '19' + oldDateFormat[i];
+        }
+      } else {
+        newDateFormat[toFormat.indexOf('YY')] = oldDateFormat[i].slice(-2);
```

Comment:
The conversion from four-digit years to two-digit years by slicing the last two digits is correct, but it's important to ensure that the `toFormat` array actually contains 'YY' before attempting to place the value in the `newDateFormat` array.